src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -294,15 +294,15 @@
          :chevron-position :left}]])))
 
 (defn page-nav-right-section-buttons
-  [id]
-  [{:icon             :i/options
-    :background-color (scroll-page/icon-color)
-    :on-press         #(rf/dispatch
-                        [:bottom-sheet/show-sheet
-                         {:content
-                          (fn []
-                            [options/community-options-bottom-sheet
-                             id])}])}])
+  [id name]
+  [{:icon                :i/options
+    :background-color    (scroll-page/icon-color)
+    :accessibility-label ((comp keyword str) "community-options-for-" name "-community")
```

Comment:
since this page only ever has one community on it at a time I think we can just have it as:
`:community-options-for-community`
A user who needs the accessibility label should already be aware of which community page they are on and it will be easier for testing this way 👍

Reply:
Amazing point :)
Thanks Jamie